packages/core/src/components/VerticalNav/VerticalNav.example.jsx

```diff
@@ -1,14 +1,23 @@
+import PropTypes from 'prop-types';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import VerticalNav from './VerticalNav';
 
+// Mock react-router example
+const Link = props => (
+  // <Link to={props.href} {...props}>{props.children}</Link>
+  <span {...props}>{props.children}</span>
```

Comment:
I feel like by having this example we're almost condoning unsemantic interactive elements. What if we just did something like this:
```js
<a {...props}><span className="really-special-link">{props.children}</span></a>
```

Reply:
Ah good point, it does look like that. I chose span arbitrarily so that you could differentiate the elements using `component` in the example, but a class will do the trick!